### DISCUSSION TITLE
pwpolicy_account_lockout_enforce issues with Sonoma

### DISCUSSION THREAD

**Discussion by danieldotweinert** [usnistgov/macos_security#310] [Q&A]
Currently having this issue with the Sonoma compliance script...as you can see the output show it as 'yes' but includes a line break and extra yes which is causing the rule to fail and make the device non compliant. Any ideas? 

Thu Sep 28 19:58:14 UTC 2023 pwpolicy_account_lockout_enforce failed (Result: yes
yes, Expected: {string: yes})

**Comment by robertgendler** [comment]
@danieldotweinert moved this to an issue
https://github.com/usnistgov/macos_security/issues/311

Closing the discussion for it.
